dom.bs

```diff
@@ -1146,7 +1146,7 @@ must return a new {{EventTarget}}.
 if this would be useful for your programs. For now, all author-created {{EventTarget}}s do not
 participate in a tree structure.</p>
 
-<p>To <dfn export>add an event listener</dfn> given an {{EventTarget}} object <var>eventTarget</var>
+<p>To <dfn export>add an event listener</dfn>, given an {{EventTarget}} object <var>eventTarget</var>
```

Comment:
The last thing needs to go on the next line due to exceeding 100 columns.

Reply:
Fixed in 3142d90.